examples/combined_ODNP.py

```diff
@@ -65,7 +65,7 @@
 dB_settings = Ep_spacing_from_phalf(
     est_phalf = config_dict['guessed_phalf'],
     max_power = config_dict["max_power"], 
-    config_dict["power_steps"] + 1, 
+    p_steps = config_dict["power_steps"] + 1, 
```

Comment:
why are you feeding +1 here? Why not just config_dict['power_steps']?

Reply:
this was pulled in from previous runs (prior to when we had thermals in a separate loop to accomodate a power axis long enough for including the thermals. - but you point out that this is no longer needed